privaterelay/management/commands/invite_monitor_waitlist.py

```diff
@@ -1,30 +1,93 @@
 from datetime import datetime
+from email.utils import parseaddr
+import logging
 
+from socketlabs.injectionapi.message.basicmessage import BasicMessage
+from socketlabs.injectionapi.message.emailaddress import EmailAddress
+
+from django.conf import settings
 from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
 
+from emails.utils import get_socketlabs_client, socketlabs_send
 from ...models import Invitations, MonitorSubscriber
 
 
+logger = logging.getLogger('events')
+
+
 class Command(BaseCommand):
     help = 'Connects to Monitor waitlist and creates invitations.'
 
+    def add_arguments(self, parser):
+        parser.add_argument('limit', nargs=1, type=int)
+
     def handle(self, *args, **options):
+        limit = options['limit'][0]
+
         monitor_waitlist = MonitorSubscriber.objects.using('monitor').filter(
             waitlists_joined__email_relay__notified=False
-        )
+        )[:limit]
+
         print("adding %s invitations ..." % monitor_waitlist.count())
+
         for invitee in monitor_waitlist:
             try:
                 Invitations.objects.get(
                     email=invitee.primary_email,
                     active=True,
                 )
-                print("%s already has an active invitation" % invitee.primary_email)
+                print(
+                    "%s already has an active invitation" %
+                    invitee.primary_email
+                )
             except Invitations.DoesNotExist:
                 print("adding %s to invitations" % invitee.primary_email)
-                Invitations.objects.create(
+                invitation = Invitations.objects.create(
                     email=invitee.primary_email,
                     active=True,
                     date_added=datetime.now(),
                     date_redeemed=None
                 )
+
+                context = {
+                    'email': invitee.primary_email,
+                    'current_domain': settings.SITE_URL,
+                }
+
+                # Send email invite
+                sl_message = BasicMessage()
+                sl_message.subject = (
+                    "You're Invited to the Private Relay Beta!"
+                )
+
+                sl_message.html_body = render_to_string(
+                    'emails/beta_invite_html_email.html',
+                    context,
+                )
+                sl_message.plain_text_body = render_to_string(
+                    'emails/beta_invite_text_email.txt',
+                    context,
+                )
+
+                relay_display_name, relay_from_address = parseaddr(
+                    settings.RELAY_FROM_ADDRESS
+                )
```

Comment:
Consider using the function `_generate_relay_From()` from the `emails/views.py` as seen [here](https://github.com/mozilla/fx-private-relay/pull/303/files#diff-2c3a9c773eb79ee1e4b1858c23620e74R201-R207). We may want to move the method to `emails/utils.py`, what do you think?

Reply:
`_generate_relay_From()` affixes `via Firefox Private Relay` to a relay email's `From:`. For the invitations, we just want our own `From:`.